SparkleLayers/SparkleLayers.pde

```diff
@@ -0,0 +1,85 @@
+import themidibus.*;
+
+OPC opc;
+RedLayer r;
+GreenLayer g;
+CloudLayer c;
+
+HashMap<Integer, Integer> faders = initFaders();
+HashMap<Integer, Integer[]> knobs = initKnobs();
+void setup()
+{
+  size(120, 300);
+
+  // Connect to the local instance of fcserver. You can change this line to connect to another computer's fcserver
+  opc = new OPC(this, "127.0.0.1", 7890);
```

Comment:
can we easily parameterize this? if yes, let's. if it requires more than a few mins of work let's do it later